go-controller/pkg/ovn/gateway_init.go

```diff
@@ -500,7 +500,7 @@ func addPolicyBasedRoutes(nodeName, mgmtPortIP string, hostIfAddr *net.IPNet) er
 		// Local gw mode needs to use DGP to do hostA -> service -> hostB
 		matchStr = fmt.Sprintf("%s.src == %s && %s.dst == %s /* inter-%s */",
 			l3Prefix, mgmtPortIP, l3Prefix, hostNet.String(), nodeName)
-		_, stderr, err = util.RunOVNNbctl("lr-policy-add", ovnClusterRouter, interNodePolicyPriority, matchStr,
+		_, stderr, err = util.RunOVNNbctl("lr-policy-add", ovnClusterRouter, fmt.Sprintf("%d", interNodePolicyPriority), matchStr,
```

Comment:
and here?

Reply:
set all the constants to strings so it is no longer necessary